### ISSUE TITLE
Sigma calculation in hyper_fit algorithm

### ISSUE THREAD

**Issue by probstj** [AlliedToasters/circle-fit#8]
I believe there is a small issue in the calculation of the sigma value used in the hyper_fit algorithm:

```
def sigma(coords, x, y, r):
    """Computes Sigma for circle fit."""
    dx, dy, sum_ = 0., 0., 0.

    for i in range(len(coords)):
        dx = coords[i][1] - x
        dy = coords[i][0] - y
        sum_ += (sqrt(dx*dx+dy*dy) - r)**2
    return sqrt(sum_/len(coords))
```

The indices of the x and y coords are switched, it should be:
`dx = coords[i][0] - x`
`dy = coords[i][1] - y`

Best regards!

**Comment by adaber** [comment]
I think you are correct, probstj.

I had implemented this algorithm in C++ a long time ago and stumbled upon this repo while searching for circle fitting in Python recently.

This code produces exactly the same values as my C++ code except for the sigma value. I tried fitting a circle through points that lie very close to the circle's outline and the sigma value calculated by the sigma method here was way off (8 instead of 0.6)  . The indices should definitely be switched.

Cheers!
